src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -1,4 +1,42 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity(name="order")
```

Comment:
W innych pull requestach była tutaj nazwa tabeli "Orders" zastosujmy to tutaj. Nazwy samego Entity nie ma sensu zmieniać